ci/run_cugraph_pyg_pytests.sh

```diff
@@ -9,7 +9,7 @@ cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../python/cugraph-pyg/cugraph_
 pytest --cache-clear --benchmark-disable "$@" .
 
 # Used to skip certain examples in CI due to memory limitations
-export CI_RUN=1
+export CI=true
```

Comment:
If we export this, it means that local executions of this script will also skip those tests. I'm not sure if that is really intended or not.

Reply:
I think the `cugraph` developers intended that. @alexbarghi-nv introduced that in https://github.com/rapidsai/cugraph/pull/4384